frontend/src/pages/project/Settings/dialogs/ImportProjectDialog.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <ff-dialog ref="dialog" header="Import Flows" confirm-label="Import Project" @confirm="confirm()" kind="danger" :disablePrimary="disabled">
+        <template v-slot:default>
+            <form class="space-y-6" @submit.prevent>
+                <div class="mt-2 space-y-2">
+                    <p class="text-sm text-gray-500">
+                        Replace current flows by uploading a new flow file. This will restart the project to pick up the new flow or credentials
+                    </p>
+                </div>
+                <ImportProjectComponents id="importSettings" v-model="parts"/>
```

Comment:
You can define a `clear()` function in `frontend/src/pages/project/components/ImportProjectComponents.vue`, then here,
add a `ref="project-components"`.

In the `@cancel` handler call `this.$refs['project-components'].clear() and it'll run the relevant functionality defined in that component

Reply:
VSCode complains if I try and use `v-model="localModelValue"` with a input of type file (in the FormRow.vue I had to add a new case for the file type)

Reply:
You don't need to change anything re: `v-model`. I'm happy to put the relevant functionality in myself, or can talk through in a huddle.